manifests/charts/istiod-remote/files/gen-istiod-remote.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: v1
 kind: ConfigMap
```

Comment:
This file should not have changed

Reply:
Yes, I have run make gen without latest changes from the upstream. Fixing now.